NotesBackEnd/tests/Notes.Api.Tests/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="1.1.17" targetFramework="net471" />
+  <package id="MSTest.TestFramework" version="1.1.17" targetFramework="net471" />

```

Comment:
With the NuGet packages, I meant these two.

Reply:
Now, I know. I installed NUnit and found this problem. I will fix it.